doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -671,13 +671,15 @@ file is a particular case of module called *library file*.
       the time it was compiled.
 
 
-   .. exn:: Require is not allowed inside a module or a module type.
+   .. exn:: Require inside a module is deprecated and strongly discouraged. You can Require a
+            module at toplevel and optionally Import it inside another one.
+            [require-in-module,deprecated]
```

Comment:
This is interpreted as three error messages. The message should be on a single line. And it should be a `.. warn::` not a `.. exn::`. And it should not include the `[require-in-module,deprecated]` which is the warning name and category and which we don't include until we fix #8125.